### DISCUSSION TITLE
Baking An Open Source Cake

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#427] [Announcements]
# Baking An Open Source Cake

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2024/06-baking-an-open-source-cake/

**Comment by Xuanwo** [comment]
I received a gift from @HerringtonDarkholme!

https://x.com/hd_nvim/status/1815423371225784665

![image](https://github.com/user-attachments/assets/857d6c1f-3721-4b10-8ef9-99d44c987e05)

**Reply by iFurySt** [reply]
Awesom!
